waiter/resources/web/sim.html

```diff
@@ -167,22 +243,46 @@ <h4>Scaler Settings</h4>
                     });
                 }
 
-                $.plot("#chart", $.map(dataSets, function(value) {
+                var totalUtilization = 0.0;
+                var totalWaste = 0.0;
+                for (var tick = 0; tick < numTicks; tick++) {
```

Comment:
Why aren't these metrics calculated in the backend with all of the other metrics?

Reply:
Done, moved the computations to `simulator.clj` and removed any metrics that needed computation in the UI.